deployment/docker/Dockerfile

```diff
@@ -28,5 +28,6 @@ RUN apt-get update \
     e2fsprogs \
     bash
 
+USER ContainerAdministrator
```

Comment:
I think this should go in the Windows Dockerfile, it'd be great if this PR is only about this change too